todos.js

```diff
@@ -35,8 +36,8 @@ app.use(
 );
 
 app.use(flash());
-app.use((req, res, next) => {
-  res.locals.store = new PgPersistence(req.session);
+app.use(async (req, res, next) => {
+  res.locals.store = new PgPersistence(await getClient(), req.session);
```

Comment:
this is dependency injection - the PgPersistence class depends on Client, and we inject an instance of Client to it via the constructor.